composer.json

```diff
@@ -23,11 +23,12 @@
         }
     ],
     "require": {
-        "php": "^7.2",
+        "php": "^7.4",
         "ext-json": "*"
     },
     "require-dev": {
-        "phpunit/phpunit": "^8.0"
+        "larapack/dd": "^1.1",
```

Comment:
Still not sure about this need as dependency. But I know that you do this quite often and expect it to be available at spatie - so I won't reject it as long as it's not conflicting.

Reply:
If it's conflicting we'll remove it!